packages/wonder-blocks-grid/util/types.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+export type GridSize = "small" | "medium" | "large";
+
+export type GridSizes = {
```

Comment:
I would call this a `GridSpec` mainly b/c `GridSizes` is too close to `GridSize` and makes me think that `type GridSize = Array<GridSize>` when it isn't.